integration-tests/src/test/java/oracle/kubernetes/operator/ItImageTool.java

```diff
@@ -0,0 +1,225 @@
+
+      // Build WebLogic Docker image using imagetool
+      buildWlsDockerImage();
+
+      // initialize test properties and create the directories
+      initialize(APP_PROPS_FILE, testClassName);
+    }
+  }
+
+  /**
+   * This method gets called before every test. It creates the result/pv root directories
+   * for the test. Creates the operator and domain if its not running.
+   *
+   * @throws Exception exception if result/pv/operator/domain creation fails
+   */
+  @BeforeEach
+  public void prepare() throws Exception {
+    if (FULLTEST) {
+      createResultAndPvDirs(testClassName);
+      String testClassNameShort = "itimage";
+      // create operator1
+      if (operator == null) {
+        Map<String, Object> operatorMap =
+            createOperatorMap(getNewSuffixCount(), true, testClassNameShort);
+        LoggerHelper.getLocal().log(Level.INFO, "Before createOperator ");
+        operator = TestUtils.createOperator(operatorMap, Operator.RestCertType.SELF_SIGNED);
+        LoggerHelper.getLocal().log(Level.INFO, "Aefore createOperator");
+        Assertions.assertNotNull(operator);
+        domainNS1 = ((ArrayList<String>) operatorMap.get("domainNamespaces")).get(0);
+        namespaceList = new StringBuffer((String) operatorMap.get("namespace"));
+        namespaceList.append(" ").append(domainNS1);
+      }
+
+      // create domain
+      if (domain == null) {
+        LoggerHelper.getLocal().log(Level.INFO,
+            "Creating WLS Domain & waiting for the script to complete execution");
+        Map<String, Object> wlstDomainMap =
+            createDomainMap(getNewSuffixCount(), testClassNameShort);
+        wlstDomainMap.put("namespace", domainNS1);
+        wlstDomainMap.put("weblogicImageTagWIT", weblogicImageTagWIT);
+        domain = TestUtils.createDomain(wlstDomainMap);
+        domain.verifyDomainCreated();
+      }
+    }
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories.
+   *
+   * @throws Exception exception
+   */
+  @AfterAll
+  public static void staticUnPrepare() throws Exception {
+    if (FULLTEST) {
+      if (namespaceList != null) {
+        tearDown(new Object() {}.getClass()
+            .getEnclosingClass().getSimpleName(), namespaceList.toString());
+      } else {
+        LoggerHelper.getLocal().log(Level.INFO, "namespaceList is null!!");
+      }
+
+      LoggerHelper.getLocal().log(Level.INFO, "SUCCESS");
+    }
+  }
+
+  /**
+   * Verify that WebLogic Docker image created by using WebLogic Image Tool is used
+   * to create WebLogic domain in Operator env.
+   * There are two ways to use the WLS Docker image created by imagetool
+   *  1. export IMAGE_NAME_WEBLOGIC = "name of your wls Docker image"
+   *     export IMAGE_TAG_WEBLOGIC = "version of the image"
+   *  2. use the values of weblogicImageName and weblogicImageTag in OperatorWIT.properties
+   *
+   * @throws Exception exception
+   */
+  @Test
+  public void testCustomImageUsed() throws Exception {
```

Comment:
Modify test to delete the domain and operator if the test completed successfully.

Reply:
fixed